<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModesWindow</class>
 <widget class="QDialog" name="ModesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1286</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Georgia</family>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="MinMaxButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>281</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">&lt;p&gt;PERSON&lt;b&gt; VS&lt;/b&gt; PERSON&lt;/p&gt;</string>
   </property>
   <property name="text">
    <string>
PERSON
VS
PERSON
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BetaAlphaButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>281</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">&lt;p&gt;PERSON
&lt;strong&gt;VS&lt;/strong&gt; 
COMPUTER&lt;/p&gt;</string>
   </property>
   <property name="text">
    <string>
PERSON
 VS
COMPUTER
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BettaAlphaIterativeButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>330</y>
     <width>281</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">&lt;p&gt;COMPUTER
&lt;strong&gt;VS&lt;/strong&gt; 
COMPUTER&lt;/p&gt;</string>
   </property>
   <property name="text">
    <string>
COMPUTER
VS
COMPUTER
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>0</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h1 style=&quot; margin-top:18px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:130pt; font-weight:700; color:##FFFFFF&quot;&gt;Choose An Algorithm&lt;/span&gt;&lt;/h1&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>0</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h1 style=&quot; margin-top:18px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:130pt; font-weight:700; color:###FFFFFF&quot;&gt;Choose Difficulty Level&lt;/span&gt;&lt;/h1&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>1261</x>
     <y>0</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>630</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>0</y>
     <width>20</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="chooseModeButton">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>20</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>High Tower Text</family>
     <pointsize>33</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">CHOOSE MODE 
font-family:Tenor Sans;</string>
   </property>
   <property name="text">
    <string>CHOOSE MODE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
